docs/application/dotnet/guides/app-management/component-port.md

```diff
@@ -0,0 +1,467 @@
+
+    ```
+    using Tizen.Applications.ComponentBased;
+    ```
+
+<a name="local"></a>
+## Managing component port
+
+To send a request from one component `ClientService.Tizen` to another `ServerService.Tizen` component, use the [Tizen.Applications.ComponentBased.ComponentPort](https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.ComponentBased.ComponentPort.html) class as follows:
+
+1.  Create a component port instance for sending component `ClientService.Tizen` as follows:
+
+    ```
+    namespace ClientService.Tizen
+    {
+        [Serializable]
+        public class Request
+        {
+            public Request(string command, int sequence, string message)
+            {
+                Command = command;
+                Sequence = sequence;
+                Message = message;
+            }
+
+            public string Command
+            {
+                get; set;
+            }
+
+            public string Message
+            {
+                get; set;
+            }
+
+            public int Sequence
+            {
+                get; set;
+            }
+        }
+
+        [Serializable]
+        public class Response
+        {
+            public Response(string command, int sequence, int result)
+            {
+                Command = command;
+                Sequence = sequence;
+                Result = result;
+            }
+
+            public string Command
+            {
+                get; set;
+            }
+
+            public int Sequence
+            {
+                get; set;
+            }
+
+            public int Result
+            {
+                get; set;
+            }
+        }
+
+        public class ClientService : ServiceComponent
+        {
+            private static string LogTag = "ClientService";
+            private static int _seq = 0;
+            private ComponentPort _port;
+
+            public override override OnCreate()
+            {
+                Log.Info("OnCreate()");
+                base.OnCreate();
+                _port = new ComponentPort("ClientService");
+                return true;
+            }
+
+            public override void OnStartCommand(AppControl appControl, bool restarted)
+            {
+                Log.Info("OnStartCommand()");
+                base.OnStartCommand(appControl, restarted);
+            }
+
+            public override void OnDestroy()
+            {
+                Log.Info("OnDestroy()");
+                base.OnDestroy();
+
+                if (_port != null)
+                {
+                    _port.Dispose();
+                    _port = null;
+                }
+            }
+        }
+    }
+    ```
+
+2. Create a component port instance for receiving component `ServerService.Tizen` as follows:
+
+    ```
```

Comment:
Global: 
Please add language identifier.

Like 
```Csharp

Reply:
Could you show me an example?
And, I think it's not needed. Because, it's a guide of .NET  application.
The developer knows the program language is CSharp.

Reply:
@hjhun  I understand, you are right, but thats part of style guide. 

https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#inline-code-blocks-with-language-identifier

Reply:
I got it. Done.

Reply:
Thats so nice of you. 
Thanks